service/scheduler.go

```diff
@@ -56,21 +56,22 @@ func (s *Scheduler) lockAndRun(db *gorm.DB, wf table.Workflow) {
 	}
 	result := db.Create(&lock)
 	if result.Error != nil {
-		fmt.Printf("something else is running this workflow %s! skip...\n", wf.ID)
+		fmt.Printf("something else is running this workflow %v! skip...\n", wf.ID)
 		return
 	}
 
 	existingLock := table.WorkflowSchedulerLock{}
 	db.First(&existingLock, wf.ID)
 
 	if existingLock.Token != token {
-		fmt.Println("something else is running this workflow %s! skip...", wf.ID)
+		fmt.Printf("something else is running this workflow %v! skip...\n", wf.ID)
```

Comment:
@schowsf guess no `fmt.Printlnf`?

Reply:
just to make it consistent with the above L59

Reply:
ok, just checked no `fmt.Printlnf` available